crates/model/src/order.rs

```diff
@@ -486,11 +486,37 @@ pub struct CancellationPayload {
     pub signing_scheme: EcdsaSigningScheme,
 }
 
-#[derive(Debug, PartialEq, Eq, Clone, Default, Deserialize, Serialize)]
+#[derive(Debug, PartialEq, Eq, Clone, Default, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct EthflowData {
     pub user_valid_to: i64,
-    pub is_refunded: bool,
+    pub refund_tx_hash: Option<H256>,
+}
+
+// We still want to have the `is_refunded` field in the JSON response to stay backwards compatible.
+// However, `refund_tx_hash` already contains all the information we need so we rather have a
+// custom `Serialize` implementation than redundant fields that could possibly contradict each other.
```

Comment:
Do we really need backward compatibility for an unreleased feature? Or is this used by the frontend?

Reply:
That's a good point. My assumption was that it was needed to not disrupt the frontend. In that case it could make sense to keep `isRefunded` until the frontend completely switches over to `refundTxHash` but drop `isRefunded` from the `openapi.yaml` so nobody else decides to depend on that flag in the meantime.

Reply:
Yes, its used by the front-end and we discussed it today in a call, that we want to have "backwards compatible" for the next week. We will then delete it in the week after that. 

Note that `isRefunded` will never really be published, we will take it with this PR (my next commit ) out of the openapi spec.